app/views/records/edit_fields/_in_works_ids.html.erb

```diff
@@ -0,0 +1,78 @@
+<%# Form UI behavior code and details;
+Code:
+  app/assets/javascripts/sufia/relationships_table.es6
+CSS:
+  table.relationships-api-enabled : Used by sufia JS app as selector to wire up the UI functionality
+  tr.new-row : The basic template row for cloning when user clicks "Add"
+  .btn-remove-row : Button to remove its parent TR from the table
+  .btn-add-row : Button to clone its parent TR and inject a new row into the table
+  input.new-form-control : Input field for client client side validation and additional features
+    .message.has-warning : Used to display UI errors related to input values and server errors
+HTML Properties:
+  table:
+    data-api-add-url : API url to call for adding a new row to this table
+    data-api-remove-url : API url to call for removing a row from this table
+    data-works-context : (parent|child)
+      this relates to the JSON returned from the server when successfully adding a new row.
+      See example API return:
+      {
+        "parent": {
+          "title":["dfdf"],
+          "path":"/concern/generic_works/9306sz37r",
+          "id":"9306sz37r"
+        },
+        "child": {
+          "title":["asdf"],
+          "path":"/concern/generic_works/2b88qc32v",
+          "id":"2b88qc32v"
+        }
+      }
+%>
+<% if f.object.class.multiple? key %>
+  <div class="form-group multi_value optional generic_work_in_works_ids managed">
+    <table class="table table-striped related-files relationships-api-enabled"
+           data-api-add-url=<%="/api/generic_works/#{f.object.model.id}/add_parent/:id"%>
+           data-api-remove-url=<%="/api/generic_works/#{f.object.model.id}/remove_parent/:id"%>
+           data-works-context="parent">
+      <thead>
+        <tr>
+          <th>Parent Work</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% curation_concern.in_works_ids.each do |parent_id| %>
+          <% parent = ::ActiveFedora::Base.find(parent_id) %>
```

Comment:
This is going to be really expensive and there's no great way to abstract it out later if the logic's in the view. Maybe `f.object.in_works.each`?